gm4_liquid_tanks/data/liquid_tanks/functions/liquid_value_update.mcfunction

```diff
@@ -0,0 +1,23 @@
+#@s = liquid tank
+#run from liquid_tanks:process
+
+#set values
+scoreboard players operation @s gm4_lt_value += @s gm4_lt_buffer
+scoreboard players set @s gm4_lt_buffer 0
+scoreboard players operation @s gm4_lt_value < @s gm4_lt_max
+
+#if value of 0, set to empty
+execute if score @s gm4_lt_value matches ..0 at @s run function liquid_tanks:empty
+
+#calculating teleport distance
+scoreboard players operation last_liquid_height gm4_lt_disp_val = @s gm4_lt_disp_val
+scoreboard players operation liquid_height_calc gm4_lt_disp_val = @s gm4_lt_value
+scoreboard players set 10 gm4_lt_disp_val 10
```

Comment:
`scoreboard players set 10 gm4_lt_disp_val 10`
Can this be done in init? It feels weird to set a constant every 16 ticks for every liquid tank